app/views/appointment_summaries/new.html.erb

```diff
@@ -0,0 +1,156 @@
+<%= form_for @appointment_summary, url: view_appointment_summaries_path do |f| %>
```

Comment:
`view_appointment_summaries_path` feels like it should just be `appointment_summary_path`?